app/main.py

```diff
@@ -1,6 +1,16 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store = {}
+
+    def inner(*args) -> Any:
```

Comment:
What if the function accepts keyword arguments?
```suggestion
    def inner(*args, **kwargs) -> Any:
```